src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import {Nav} from "./../../components/Nav/Nav";
+import Hero_Img from './../../assets/hero2.jpg';
+import {Card} from './../../components/Card/Card';
```

Comment:
One suggestion You can use an index.js file and add all your exports here and you can simply import all your component from a single file.

Reply:
Thanks for the suggestion.
I will implement this.